credentials.cipher

```diff
@@ -0,0 +1,4 @@
+Salted__aOS�_�Pu"��&!�(>�x��)���<ɕ
```

Comment:
well, it was recommended as "the" way to integrate circleci here - https://github.com/circleci/encrypted-files (linked from https://circleci.com/docs/environment-variables/#keeping-encrypted-environment-variables-in-source-code) - but we could put them directly into circle i guess?